### DISCUSSION TITLE
Clarify "exclusive property"

### DISCUSSION THREAD

**Discussion by onpon4** [4ian/GDevelop#2374] [Archive] [answered]
I just want to point out this minor problem: license.txt says that the name and logo of Game Develop are "exclusive property of Florian Rival". This should be clarified. It sounds to me like it's saying that these are trademarks. If that's what it's supposed to say, it should say so explicitly, and it would also be preferable if it pointed to a trademark policy.

**Comment by 4ian** [comment]
Hi onpon4,

Thanks for pointing this issue! :) These are not explicitly registered trademark: I just want to make clear that even if all the source code is free software (GPL/LGPL or zlib/png license, depending on the part of the software), the logo is not under one of these license and should be consider as something that is protected by author's right.
In fact, I just want to avoid having forks or derivate work created from Game Develop with the same name :) And also to prevent anyone from using the same logo for another project which could be very very misleading!
So nothing fancy in any case, let me know if you find a better sentence or a better text to clarify it. ;) (Or let me know if I'm not clear enough ^^)

**Comment by onpon4** [comment]
You might want to consider registering a trademark, then. If you don't have a trademark registered, all you have is copyright, which doesn't cover names and isn't designed for the job. Given the simplicity of it, it probably doesn't cover the Game Develop logo, either. So what you have right now is probably an empty statement.

Disclaimer: I'm not a lawyer. ;)

**Comment by Silver-Streak** [comment]
At this point, since it has been posted on Itch.io and other locations, it would qualify for Unregistered trademark, if there is a desire to claim trademark on it at all. 

This already provides a lot of protections in the US, but may not in other nations. https://en.wikipedia.org/wiki/Unregistered_trademark

Disclaimer: I am also not a lawyer.

**Reply by 4ian** [reply, marked as answer]
Thanks, this is an interesting concept. I've registered the trademark in some countries and will continue to expand it.
